assets/src/components/v2/departures/departure_row.tsx

```diff
@@ -9,15 +9,20 @@ type DepartureRow = {
   route: Pill;
   headsign: Destination;
   times_with_crowding: TimeWithCrowding[];
+  direction_id: 0 | 1;
+  showBoldDivider: boolean;
```

Comment:
nit: something like `isBeforeDirectionSplit` I think would better express what this indicates, without being as prescriptive about how it should be presented.